piwebapi_samples/Data Analysis/output.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" ?>
```

Comment:
Is this a test output file? If so, probably shouldn't be checked in. 
Looks like a `junit` file. Usually we put those in a `/junit` folder, and the `/junit` folders are automatically ignored by our `.gitignore`.

Reply:
Yes, it's a test output file. I can remove it or put it under a separate folder.

Reply:
I'd put the test output into a `junit` folder, then that folder will get ignored by git by default.

Reply:
I have deleted this file from the source control entirely in the latest commit. It was generated because I ran the tests locally